tachyon/math/polynomials/univariate/univariate_polynomial.h

```diff
@@ -294,6 +295,26 @@ class Copyable<math::UnivariatePolynomial<Coefficients>> {
   }
 };
 
+template <typename Coefficients>
+class RapidJsonValueConverter<math::UnivariatePolynomial<Coefficients>> {
+  public:
+    template <typename Allocator>
+  static rapidjson::Value From(const math::UnivariatePolynomial<Coefficients>& value, Allocator& allocator) {
+    rapidjson::Value object(rapidjson::kObjectType);
+    AddJsonElement(object, "coefficients", value.coefficients().coefficients(), allocator);
+    return object;
+  }
+
+  static bool To(const rapidjson::Value& json_value, std::string_view key,
```

Comment:
```suggestion
  [[nodiscard]] static bool To(const rapidjson::Value& json_value, std::string_view key,
```

Reply:
Is it necessary to include the `nodiscard` keyword in utility functions? In my opinion, I'd prefer not to include it for user's convenience.

Reply:
Make sense, cuz it's not as critical as core logic